products/constants.py

```diff
@@ -0,0 +1,9 @@
+"""
+Contains values which dont change during runtime
+"""
+# Templates
+HOMEPAGE = "homepage.html"
```

Comment:
I see only one usage for this ? Do we need this as a constant?

Reply:
Can you check if the below-mentioned constants are also used once?

Reply:
We don't need these constants, I have removed them.